twitch-chat-reader/src/listener.py

```diff
@@ -88,7 +88,12 @@ async def read(self) -> None:
 
         while True:
             data = await self.reader.read(1024)
-            messages = data.decode()
+            try:
+                messages = data.decode()
```

Comment:
while monitor another channel data didn't get decode with UTF-8.. should fix this better, but this will at least log the bad message.